src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -88,17 +89,26 @@ public void actionPerformed(ActionEvent arg0) {
     chkParallel.addActionListener(new ParallelPullToggleListener());
 
     chkTrackingConsent.setSelected(BriefcasePreferences.getBriefcaseTrackingConsentProperty());
-    chkTrackingConsent.addActionListener(new TrackingConsentToggleListener(parentWindow.briefcaseAnalytics));
+    chkTrackingConsent.addActionListener(__ -> {
+      if (chkTrackingConsent.isSelected())
+        analytics.enableTracking();
+      else
+        analytics.disableTracking();
+      BriefcasePreferences.setBriefcaseTrackingConsentProperty(chkTrackingConsent.isSelected());
+
+    });
 
     chkStorePasswordsConsent.setSelected(BriefcasePreferences.getStorePasswordsConsentProperty());
     chkStorePasswordsConsent.addActionListener(e -> {
-      if (e.getSource() == chkStorePasswordsConsent) {
-        boolean isSelected = chkStorePasswordsConsent.isSelected();
-        BriefcasePreferences.setStorePasswordsConsentProperty(isSelected);
-        if (!isSelected) {
-          PullTransferPanel.PREFERENCES.remove(BriefcasePreferences.PASSWORD);
-          PushTransferPanel.PREFERENCES.remove(BriefcasePreferences.PASSWORD);
-        }
+      boolean isSelected = chkStorePasswordsConsent.isSelected();
+      if (isSelected)
```

Comment:
Come on, brother, I know you prefer this shorter style:
`analytics.event("Settings", "Save passwords", isSelected ? "Enable" : "Disable", null);`

Reply:
Good catch! Thanks!